recipes/scikit-posthocs/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020 Maksim Terpilowski
```

Comment:
Since the license file is in the source distribution now, you can delete this manually packaged copy. After that I'm happy to merge.

Reply:
Done. Thanks again!